### DISCUSSION TITLE
Model listing Error

### DISCUSSION THREAD

**Discussion by DawoodTouseef** [open-webui/open-webui#4344] [General]
# Bug Report

## Description

**Bug Summary:**
The user with the role "user" is unable to list any models in the Open WebUI. When attempting to view models, the interface displays "No results found."

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

**Expected Behavior:**
The user should be able to see a list of models available for their role. The models should be displayed based on the filtering logic applied.

**Actual Behavior:**
The interface displays "No results found," and no models are listed for users with the "sb" role. Other user roles do not experience this issue.

## Environment

- **Open WebUI Version:** [e.g., 0.3]
- **Ollama (if applicable):** [e.g., 0.1.30, 0.1.32-rc1]

- **Operating System:** Ubuntu 20.04

**Comment by av** [comment]
I think I'm experiencing this or similar issue on 0.3.11 with the following config:

```json
{
  "openai": {
    "api_base_urls": [
      "http://litellm:4000/v1"
    ],
    "api_keys": [
      "sk-litellm"
    ],
    "enabled": true
  }
}
```

Open WebUI tries to reach for a not present Ollama endpoint and doesn't list models from the OpenAI endpoints

**Comment by saffanalvy** [comment]
I just do a 'ollama pull llama3.1', put your desired model name in the command and update ollama. If it still does not show then give a quick restart to your ollama and webui docker container. Keep your open webui and especially ollama up to date. You need to know that after the ollama pull command a pop-up will appear in the taskbar to update ollama in windows. Keep your webui open and after the update your model will show in the webui. This fixes my problem.
